### PR TITLE
set color of tick-labels separately

### PR DESCRIPTION
**Pull request by SamSchott**

### CODE REVIEW

File: pyqtgraph/graphicsItems/PlotCurveItem.py [hunk 1]
```diff
@@ -502,7 +502,11 @@ def paint(self, p, opt, widget):
             p.setPen(sp)
             p.drawPath(path)
         p.setPen(cp)
-        p.drawPath(path)
+        if self.fillPath is not None:
+            p.drawPath(self.fillPath)
+        else:
+            p.drawPath(path)
+        profiler('drawPath')
```
Comment (j9ac9k) [inline pyqtgraph/graphicsItems/PlotCurveItem.py]: Do we need two calls to `profiler('drawPath')`}

Comment (SamSchott) [reply pyqtgraph/graphicsItems/PlotCurveItem.py]: No, we don't. Something got mixed up.

**Comment by j9ac9k** [comment]
Looks like the CI is having a cow:

```
AttributeError: 'AxisItem' object has no attribute '_textPen'
Traceback (most recent call last):
  File "/home/vsts/work/1/s/pyqtgraph/graphicsItems/AxisItem.py", line 556, in paint
    self.drawPicture(painter, *specs)
  File "/home/vsts/work/1/s/pyqtgraph/graphicsItems/AxisItem.py", line 1063, in drawPicture
    p.setPen(self.textPen())
  File "/home/vsts/work/1/s/pyqtgraph/graphicsItems/AxisItem.py", line 407, in textPen
    if self._textPen is None:
```

Need to add `self._textPen` to the `__init__` method?

File: pyqtgraph/graphicsItems/PlotCurveItem.py (hunk 1 above)
Comment (j9ac9k) [reply pyqtgraph/graphicsItems/PlotCurveItem.py]: no problem!

**Comment by SamSchott** [comment]
Somehow, code from pull requests #836, #838 and #840 got mixed up in this. Let me correct that.

**Comment by SamSchott** [comment]
> Need to add `self._textPen` to the `__init__` method?

Should be fixed now.

**Comment by j9ac9k** [comment]
closing and opening to trigger CI

**Comment by j9ac9k** [comment]
CI passed across the board, merging.

**Pull request merged**